support/get-resources.sh

```diff
@@ -23,7 +23,7 @@ cat << EOF > ${work_file}
     },
     {
       "id": "release",
-      "cmd": "oc get secret manifest -n $(oc get ns | egrep '^(openshift-|)webapp' | awk '{print $1}') -o template='{{(.data.generated_manifest | base64decode) }}' | jq '.version' -r",
+      "cmd": "oc get dc/tutorial-web-app -n redhat-rhmi-solution-explorer -o json | jq '.spec.template.spec.containers[].env[] | select(.name == \"INTEGREATLY_VERSION\").value' -r",
```

Comment:
I think I missed this one during my previous review, but this command line is specific to 4.x and breaks compatibility with 3.x.

Reply:
Hmm, not sure the best way to differentiate between 3.x and 4.x here. Any suggestions?